docs/content/how-to-guides/configure-reactions/configure-drasi-result-reaction/_index.md

```diff
@@ -3,8 +3,137 @@ type: "docs"
 title: "Configure a Drasi Result Reaction"
 linkTitle: "Configure a Drasi Result Reaction"
 weight: 60
-toc_hide: true
-hide_summary: true
 description: >
     Learn how to configure a Drasi Result Reaction
----
\ No newline at end of file
+---
+
+The Drasi Result Reaction allows users to retrieve the current result set, or the result set at a specified timestamp, for a particular Continuous Query. This reaction provides an endpoint accessible via HTTP GET requests.
+
+## Requirements
+On the computer from where you will create the Drasi Debug Reaction, you need to install the following software:
+- [Drasi CLI](/reference/command-line-interface/) 
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+
+You should also be proficient in creating HTTP requests using tools such as [curl](https://curl.se/) or [Postman](https://www.postman.com/).
+
+## Creating the Reaction
+To create a Reaction, execute the `drasi apply` command as follows:
+
+```text
+drasi apply -f my-reaction.yaml -n drasi-namespace
```

Comment:
I don't think we need to mention the namespace option in every example, if it is covered in the CLI reference?
@agentofreality thoughts?

Reply:
I can remove the `-n` from the snippet, and add a short sentence that talks about the namespace in the description below